neurolib/models/multimodel/builder/fitzhugh_nagumo.py

```diff
@@ -0,0 +1,179 @@
+"""
+FitzHugh–Nagumo model.
+
+Main references:
+    FitzHugh, R. (1955). Mathematical models of threshold phenomena in the
+    nerve membrane. The bulletin of mathematical biophysics, 17(4), 257-278.
+
+    Nagumo, J., Arimoto, S., & Yoshizawa, S. (1962). An active pulse
+    transmission line simulating nerve axon. Proceedings of the IRE, 50(10),
+    2061-2070.
+
+Additional reference:
+    Kostova, T., Ravindran, R., & Schonbek, M. (2004). FitzHugh–Nagumo
+    revisited: Types of bifurcations, periodical forcing and stability regions
+    by a Lyapunov functional. International journal of bifurcation and chaos,
+    14(03), 913-925.
+"""
+
+import numpy as np
+from jitcdde import input as system_input
+
+from ..builder.base.network import Network, Node
+from ..builder.base.neural_mass import NeuralMass
+
+DEFAULT_PARAMS = {
+    "alpha": 3.0,
+    "beta": 4.0,
+    "gamma": -1.5,
+    "delta": 0.0,
+    "epsilon": 0.5,
+    "tau": 20.0,
+    "ext_input_x": 1.0,
+    "ext_input_y": 0.0,
+}
+
+
+class FitzHughNagumoMass(NeuralMass):
+    """
+    FitzHugh-Nagumo neural mass.
+    """
+
+    name = "FitzHugh-Nagumo mass"
+    label = "FHNmass"
+
+    num_state_variables = 2
+    num_noise_variables = 2
+    coupling_variables = {0: "x", 1: "y"}
+    state_variable_names = ["x", "y"]
+    required_params = [
+        "alpha",
+        "beta",
+        "gamma",
+        "delta",
+        "epsilon",
+        "tau",
+        "ext_input_x",
+        "ext_input_y",
+    ]
+    required_couplings = ["network_x", "network_y"]
+
+    def __init__(self, params=None, seed=None):
+        super().__init__(params=params or DEFAULT_PARAMS, seed=seed)
+
+    def _initialize_state_vector(self):
+        """
+        Initialize state vector.
+        """
+        np.random.seed(self.seed)
+        self.initial_state = (0.05 * np.random.uniform(-1, 1, size=(self.num_state_variables,))).tolist()
+
+    def _derivatives(self, coupling_variables):
+        [x, y] = self._unwrap_state_vector()
+
+        d_x = (
+            -self.params["alpha"] * x ** 3
+            + self.params["beta"] * x ** 2
+            + self.params["gamma"] * x
+            - y
+            + coupling_variables["network_x"]
+            + system_input(self.noise_input_idx[0])
+            + self.params["ext_input_x"]
+        )
+
+        d_y = (
+            (x - self.params["delta"] - self.params["epsilon"] * y) / self.params["tau"]
+            + coupling_variables["network_y"]
+            + system_input(self.noise_input_idx[1])
+            + self.params["ext_input_y"]
+        )
+
+        return [d_x, d_y]
+
+
+class FitzHughNagumoNetworkNode(Node):
```

Comment:
```suggestion
class FitzHughNagumoNode(Node):
```
To avoid confusion with `Network`